app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,65 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.JsonTask;
+import com.codesoom.assignment.application.TaskApplicationService;
+import com.codesoom.assignment.application.TaskJsonTransfer;
+import com.codesoom.assignment.domain.Task;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/tasks")
+public class TaskController {
+    TaskApplicationService taskApplicationService = new TaskApplicationService();
+    TaskJsonTransfer transfer = new TaskJsonTransfer();
+
+    @GetMapping
+    public String getAllTasks() throws UnknownException {
+        return transfer.taskListToJson(taskApplicationService.getAllTasks())
+            .orElseThrow(UnknownException::new);
+    }
```

Comment:
`json`문자열로 바꿔서 문자열을 반환하고 있는 것 같아요. 다음과 같이 쓰는건 어떨까요?

```suggestion
    @GetMapping
    public List<Task> getAllTasks() throws UnknownException {
        return taskApplicationService.getAllTasks()
    }
```

Reply:
https://github.com/CodeSoom/spring-week2-assignment-1/pull/18/commits/06eb7b2796f431bed3053749af73cc15c2eb58c2 에서 변경하였습니다!! 이러면 orElseThrow도 없앨 수 있겠네요!!